### DISCUSSION TITLE
Slimproto/Squeezelite Display Help?

### DISCUSSION THREAD

**Discussion by wryandginger** [music-assistant/support#1312] [Feature requests and ideas]
Is there any way of adjusting the way the metadata is displayed on Slimproto devices?

I have an ESP32-A1S Audio Kit with SSD1306 display (128 x 64 pixels) running SqueezeliteESP32. My display is working, but Music Assistant v2 (beta 35) only shows Artist - Title on line 1 and then a VU meter below the text. This isn't terrible, but the text doesn't scroll so I only ever see the artist and never the title of the track. Internet radio only displays the station name, despite the add-on displaying Artist and Track inside of Home Assistant.

I don't have this issue connecting to a LMS server and the issue doesn't happen when I connect directly to the player via C-Spot/Bluetooth/Airplay. LMS also gives several options for displaying/formatting metadata.

Is this just a current limitation of Music Assistant or am I missing a setting to make the metadata scroll or even just reverse the title-artist format?

**Comment by OzGav** [comment]
Likely a current limitation. Display support was only just added. Not sure how long until this can be incorporated but I’ll let you know what I find out

edit: added this to an existing issue https://github.com/music-assistant/hass-music-assistant/issues/1306

**Reply by marcelveldt** [reply]
I can extend the support later but first prio is getting a stable foundation and release 2.0 stable. 
Once that is doen I will focus on these small details and extra features. That it works in LMS is no guarantee that it works in MA, LMS is being developed for 30 years already while MA only for 3 years and does not cover the full LMS feature set (and it shouldnt imo).

That said. If you want a temporary solution, you can add the player as airplay to MA, then it should work already

**Reply by wryandginger** [reply]
No worries, that's why I put it as a feature request and not a bug fix. I really appreciate the work on this. 
I was mostly scratching my head because Squeezelite-esp32 firmware lets you specify how the metadata is formatted in the NVS editor, so I was testing with LMS to see if it was my syntax. 

Again, huge thanks. Even the display support currently is a massive leap forward. In MA 1.8 I was scraping metadata from the event trigger as a text sensor, then displaying it via ESPHome on a separate display. Since beta 35, that mostly is redundant and saves me a ton of time in the future not having to rebuild those sensors for MA 2.0.
